config/locales/interface/output_elements/en_import_export.yml

```diff
@@ -33,3 +33,11 @@ en:
       title: Present energy transit flows
       short_description: 
       description:
+    space_requirements_transit_flows:
+      title: Land use of import infrastructure per carrier
+      short_description:
+      description: |
+        This chart presents the imported volume per energy carrier and the 
+        future space requirements of import facilities to process the imported
+        volume. This gives an indication of the spatial impact to facilitate transit
+        flows (import and export) of energy carriers. 
```

Comment:
Import is not necessarily only for transit flows: import might also just be for domestic use. More importantly though, I would expect to read here what components included in the land use. Can we clarify this?

Reply:
I have altered the description:

-  further specified what is included in the required space
- rephrased impact of required space for facilitating import, rather than facilitating transit flows

Reply:
🙌🏻